Sources/ArcGISToolkit/Components/Offline/PreplannedMapModel.swift

```diff
@@ -43,6 +46,11 @@ class PreplannedMapModel: ObservableObject, Identifiable {
     /// A Boolean value indicating if a user notification should be shown when a job completes.
     let showsUserNotificationOnCompletion: Bool
     
+    /// The file size of the preplanned map area.
+    var directorySize: Int {
+        FileManager.default.sizeOfDirectory(at: mmpkDirectory)
+    }
```

Comment:
Calculating the size could be time consuming, definitely not something to do on-demand, especially during a view update. Instead, it would be better to calculate the size as part of an asynchronous operation (inside of `downloadPreplannedMapArea()`?).